pkgs/applications/video/subdl/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation rec {
+  name = "subdl";
```

Comment:
It depends on reviewer, but some would only accept unversioned package with a date suffix in `name`, e.g. `subdl-2017-12-30`.

Reply:
Indeed, please put a version or at least date at the end. Otherwise, nix-env will never be able to update it.